cli/module_build_test.go

```diff
@@ -32,8 +32,8 @@ func createTestManifest(t *testing.T, path string) {
     }
   ],
   "build": {
-    "setup": "",
-    "build": "make build",
+    "setup": "./setup.sh",
+    "build": "echo build step msg",
```

Comment:
This is a test file -- I'm not sure it will guide users down the wrong path if we test echo instead of make here. 

I'm not passionate about this so if you care about creating a mock Makefile for the test, I will, just want to double check before I do

Reply:
Just made the test create a makeflle in [80f5f1e](https://github.com/viamrobotics/rdk/pull/3572/commits/80f5f1e774595858c29b44470f13cfe2db81f9cc)